exporter/views.py

```diff
@@ -13,6 +13,10 @@
 
 @csrf_exempt
 def exporter_start(request):
+    """
+    Plans (send messages to a worker) the export of collection from kingfisher-process.
+    Expects {"collection_id": <id>} in request body.
```

Comment:
```suggestion
    Adds a message to a queue to export files from a collection in Kingfisher Process.

    Expects ``{"collection_id": <id>}`` in the request body.
```

Reply:
Actually, reading the code, I'm pretty sure spider and job_id need to be set. That's what the worker expects.